src/teleport.cpp

```diff
@@ -70,19 +70,24 @@ void Teleport::addThing(Thing* thing)
 	return addThing(0, thing);
 }
 
-bool Teleport::checkInfinityLoop(Tile* destTile)
+bool Teleport::isHuge(const Tile* destTile)
```

Comment:
There is no reason to change the name of the method, especially if it's not a very descriptive name